config/production.json

```diff
@@ -25,10 +20,5 @@
 	"metrics.runtime.enableCPUMetrics": true,
 	"metrics.runtime.enableMemMetrics": true,
 	"metrics.runtime.enableGCMetrics": true,
-	"metrics.runtime.collectInterval": 1000,
-
-	"http.port": 4000,
-
-	"tchannel.serviceName": "my-gateway",
```

Comment:
Do we no longer need these?

Reply:
these are mandatory but the default values are "never useful". we will need to properly configure edge-gateway and now it will just panic instead of using a "useless default value".